components/standardizers/src/main/java/org/datacleaner/result/renderer/CountryStandardizationResultHtmlRenderer.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.result.renderer;
+
+import java.util.Collection;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang.StringEscapeUtils;
+import org.datacleaner.api.Provided;
+import org.datacleaner.api.Renderer;
+import org.datacleaner.api.RendererBean;
+import org.datacleaner.api.RendererPrecedence;
+import org.datacleaner.beans.standardize.CountryStandardizationResult;
+import org.datacleaner.result.AnnotatedRowsResult;
+import org.datacleaner.result.html.DrillToDetailsBodyElement;
+import org.datacleaner.result.html.HtmlFragment;
+import org.datacleaner.result.html.SimpleHtmlFragment;
+
+@RendererBean(HtmlRenderingFormat.class)
+public class CountryStandardizationResultHtmlRenderer implements Renderer<CountryStandardizationResult, HtmlFragment> {
+
+    private final AtomicInteger _elementCounter = new AtomicInteger(0);
```

Comment:
This should not be a field since that makes it shared between many renderings. I would simply make it a local variable of the render() method.